README.md

```diff
@@ -99,6 +99,36 @@ The custom hadoop configurations can be passed in with the `--hadoopConfig [cust
 The config in custom hadoop config file will override the default hadoop configurations. For an example
 of a custom hadoop config file, see [hadoop.xml](https://xtable.apache.org/docs/fabric#step-2-translate-source-table-to-delta-lake-format-using-apache-xtable-incubating).
 
+# New file type spotless automatic formatting and modification operation
+
+When a new type of file is merged into the warehouse, it is possible that the `spotless-maven-plugin` will not add the license header by default, and when it is detected by ci and prompted to add the `license header`, what we have to do is to modify the project. The modified part of `pom.xml` under the root path is as follows:
```

Comment:
```suggestion
When a new type of file is added to the repo, it is possible that the `spotless-maven-plugin` will not add the license header by default. This is detected by the `License Check` GitHub Action. When this happens the developer should must modify part of `pom.xml` under the root path is as follows:
```

Reply:
@wuchunfu here are a few reasons for my suggestion:
1. Warehouse is a commonly used term in the data space for a Data Warehouse (like Snowflake and BigQuery). It would be best to make sure we avoid confusion.
2. Adding the name of the check in CI can help people know what to look for.